README.md

```diff
@@ -40,14 +40,16 @@ Questions can be asked by raising an `Issue`.
   git checkout -b <name of new branch>
 ```
 
-- Make changes in your repo by opening the README.md file and adding your full name along with a link to your github profile
+<!-- - Make changes in your repo by opening the README.md file and adding your full name along with a link to your github profile -->
 
-```
-  open ReadMe. md
-  On a new line type in ..
-  Full Name [github.com/githubUsername]
-```
-- Stage your changes 
+- Open the file README. md
```

Comment:
I think the space in the file name might confuse people. Also could wrap it with inline code block

From 

README. md

To 

`README.md`

Reply:
Oh funny enough i did not even notice